packages/edge-demo-site/app/entry.client.tsx

```diff
@@ -0,0 +1,22 @@
+import { RemixBrowser } from '@remix-run/react'
```

Comment:
All the files in the edge-demo site folder are standard files that a Remix demo has.